src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -101,14 +101,14 @@ class SpanDetail extends React.Component<Props, State> {
       field: ['transaction', 'id', 'trace.span'],
       sort: ['-id'],
       query: `event.type:transaction trace:${traceID} trace.parent_span:${spanID}`,
-      project: organization.features.includes('global-views')
+      project: (organization.features.includes('global-views')
         ? [ALL_ACCESS_PROJECTS]
-        : [Number(event.projectID)],
+        : [Number(event.projectID)]) as number[] | undefined,
```

Comment:
it looks like that this will be always an array... it can be only of type `number[] `. I would say that it could also be of type  NaN[], but I think typescript doesn't have this type yet... it's being discussed https://github.com/Microsoft/TypeScript/issues/28682

Reply:
This should always be `number[]`.

Reply:
This field is deleted a few lines down so we have to either define the type of the object or quickly hack in the type of this property